.github/workflows/matrix.js

```diff
@@ -57,17 +57,15 @@ matrix.addAxis({
   values: [
     '8.4',
     '9.0',
-    '9.1',
-    '9.2',
-    '9.3',
-    '9.4',
-    '9.5',
```

Comment:
Why removing 9.x?

Reply:
faster tests. 9.5 was EOL  Feb 11 2021
We still test 9.6 which is the only sane version 9 someone should be running.

Reply:
The number of versions here does not affect test duration. We sample a few versions randomly anyways

Reply:
Still there's no real reason to test them, I'd rather test newer versions.

Reply:
We do have code like `&& connection.haveMinimumServerVersion(ServerVersion.v9_2)` and `&& assumeVersion.getVersionNum() >= ServerVersion.v9_4.getVersionNum()`, so testing with various 9.2 versions make sense.